diff -up cmake-4.0.0-rc4/Source/cmGeneratorTarget_IncludeDirectories.cxx.3~ cmake-4.0.0-rc4/Source/cmGeneratorTarget_IncludeDirectories.cxx
--- cmake-4.0.0-rc4/Source/cmGeneratorTarget_IncludeDirectories.cxx.3~	2025-03-17 13:50:57.577327710 +0100
+++ cmake-4.0.0-rc4/Source/cmGeneratorTarget_IncludeDirectories.cxx	2025-03-17 13:54:49.268987400 +0100
@@ -163,7 +163,7 @@ void processIncludeDirectories(cmGenerat
     for (std::string& entryInclude : entry.Values) {
       if (fromImported && !cmSystemTools::FileExists(entryInclude)) {
         tgt->GetLocalGenerator()->IssueMessage(
-          MessageType::FATAL_ERROR,
+          MessageType::AUTHOR_WARNING,
           cmStrCat(
             "Imported target \"", targetName,
             "\" includes non-existent path\n  \"", entryInclude,
@@ -173,7 +173,11 @@ void processIncludeDirectories(cmGenerat
             "* An install or uninstall procedure did not complete "
             "successfully.\n"
             "* The installation package was faulty and references files it "
-            "does not provide.\n"));
+            "does not provide.\n"
+            "* The installation package has intentionally been split into "
+            "several subpackages, and you haven't installed the one "
+            "containing ", entryInclude, ". This package may or "
+            "may not be required to build this package.\n"));
         return;
       }
 
