diff -up cmake-4.0.0-rc4/Source/cmLocalGenerator.cxx.4~ cmake-4.0.0-rc4/Source/cmLocalGenerator.cxx
--- cmake-4.0.0-rc4/Source/cmLocalGenerator.cxx.4~	2025-03-17 13:55:17.850090427 +0100
+++ cmake-4.0.0-rc4/Source/cmLocalGenerator.cxx	2025-03-17 13:56:34.624192288 +0100
@@ -2508,7 +2508,7 @@ void cmLocalGenerator::AddFeatureFlags(s
                  (targetType == cmStateEnums::MODULE_LIBRARY));
 
   if (target->GetLinkInterfaceDependentBoolProperty(
-        "POSITION_INDEPENDENT_CODE", config)) {
+        "POSITION_INDEPENDENT_CODE", config) && (flags.find("-fPIC") == std::string::npos)) {
     this->AddPositionIndependentFlags(flags, lang, targetType);
   }
   if (shared) {
