--- cmake-3.10.0-rc4/Source/cmLocalGenerator.cxx.0003~	2017-11-05 23:59:00.347294038 +0100
+++ cmake-3.10.0-rc4/Source/cmLocalGenerator.cxx	2017-11-05 23:59:39.022377362 +0100
@@ -1834,7 +1834,8 @@ void cmLocalGenerator::AddPositionIndepe
     flagsVar += "_COMPILE_OPTIONS_PIC";
     picFlags = this->Makefile->GetSafeDefinition(flagsVar);
   }
-  if (picFlags) {
+  // Don't allow -fPIE to override -fPIC
+  if (picFlags && flags.find("-fPIC")==std::string::npos) {
     std::vector<std::string> options;
     cmSystemTools::ExpandListArgument(picFlags, options);
     for (std::string const& o : options) {
