diff -up cmake-3.29.2/Source/cmExportFileGenerator.cxx.2~ cmake-3.29.2/Source/cmExportFileGenerator.cxx
--- cmake-3.29.2/Source/cmExportFileGenerator.cxx.2~	2024-04-21 12:43:54.368256893 +0200
+++ cmake-3.29.2/Source/cmExportFileGenerator.cxx	2024-04-21 12:45:33.561357717 +0200
@@ -1339,7 +1339,7 @@ void cmExportFileGenerator::GenerateImpo
         "    foreach(_cmake_file IN LISTS "
         "\"_cmake_import_check_files_for_${_cmake_target}\")\n"
         "      if(NOT EXISTS \"${_cmake_file}\")\n"
-        "        message(FATAL_ERROR \"The imported target "
+        "        message(WARNING \"The imported target "
         "\\\"${_cmake_target}\\\" references the file\n"
         "   \\\"${_cmake_file}\\\"\n"
         "but this file does not exist.  Possible reasons include:\n"
@@ -1348,6 +1348,14 @@ void cmExportFileGenerator::GenerateImpo
         "* The installation package was faulty and contained\n"
         "   \\\"${CMAKE_CURRENT_LIST_FILE}\\\"\n"
         "but not all the files it references.\n"
+	"* The installation package was intentionally split into smaller\n"
+	"  components, and \\\"${_cmake_file}\\\"\n"
+	"  is part of an optional subpackage. This is generally harmless\n"
+	"  and known to happen on e.g. the static libraries when only\n"
+	"  shared libraries are being used - but if you see an actual\n"
+	"  error after seeing this message, you might want to make sure\n"
+	"  the optional file is installed by running\n"
+	"  sudo dnf --refresh install \\\"${_cmake_file}\\\"\n"
         "\")\n"
         "      endif()\n"
         "    endforeach()\n"
