diff -up cmake-3.13.0/Source/cmLocalGenerator.cxx.0003~ cmake-3.13.0/Source/cmLocalGenerator.cxx
--- cmake-3.13.0/Source/cmLocalGenerator.cxx.0003~	2018-11-23 23:43:58.493477651 +0100
+++ cmake-3.13.0/Source/cmLocalGenerator.cxx	2018-11-23 23:47:15.708219133 +0100
@@ -1873,7 +1873,8 @@ void cmLocalGenerator::AddPositionIndepe
     flagsVar += "_COMPILE_OPTIONS_PIC";
     picFlags = this->Makefile->GetSafeDefinition(flagsVar);
   }
-  if (!picFlags.empty()) {
+  // Don't allow -fPIE to override -fPIC
+  if (!picFlags.empty() && flags.find("-fPIC")==std::string::npos) {
     std::vector<std::string> options;
     cmSystemTools::ExpandListArgument(picFlags, options);
     for (std::string const& o : options) {
