diff -up cmake-3.31.0/Source/cmFindCommon.cxx.1~ cmake-3.31.0/Source/cmFindCommon.cxx
--- cmake-3.31.0/Source/cmFindCommon.cxx.1~	2024-11-21 19:37:23.383587082 +0100
+++ cmake-3.31.0/Source/cmFindCommon.cxx	2024-11-21 19:43:00.156423278 +0100
@@ -454,6 +454,12 @@ void cmFindCommon::ComputeFinalPaths(Ign
                                          this->SearchPaths);
   }
 
+  // Add Qt6 paths
+  std::vector<std::string> qt6Modules;
+  cmSystemTools::GlobDirs("@QT6DIR@/lib/cmake/*", qt6Modules);
+  for(std::string const &q6m : qt6Modules)
+    this->SearchPaths.push_back(q6m);
+
   // Expand list of paths inside all search roots.
   this->RerootPaths(this->SearchPaths, debugBuffer);
 
