diff -up cmake-4.4.0/Source/cmPolicies.cxx.omv~ cmake-4.4.0/Source/cmPolicies.cxx
--- cmake-4.4.0/Source/cmPolicies.cxx.omv~	2026-07-10 16:28:03.172686810 +0200
+++ cmake-4.4.0/Source/cmPolicies.cxx	2026-07-10 16:28:03.191045085 +0200
@@ -330,13 +330,15 @@ bool cmPolicies::ApplyPolicyVersion(cmMa
       minorVer = 5;
       patchVer = 0;
     } else {
-      mf->IssueMessage(MessageType::FATAL_ERROR,
-                       "Compatibility with CMake < 3.5 has been removed "
-                       "from CMake.\n" ADVICE_UPDATE_VERSION_ARGUMENT "\n"
-                       "Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try "
-                       "configuring anyway.");
-      cmSystemTools::SetFatalErrorOccurred();
-      return false;
+      mf->IssueDiagnostic(
+        cmDiagnostics::CMD_DEPRECATED,
+        "Compatibility with CMake < 3.5 has been removed "
+        "from CMake.\n" ADVICE_UPDATE_VERSION_ARGUMENT "\n"
+        "Assuming -DCMAKE_POLICY_VERSION_MINIMUM=3.5 and "
+        "configuring anyway.");
+      majorVer = 3;
+      minorVer = 5;
+      patchVer = 0;
     }
   } else if (majorVer == 3 && minorVer < 10 && warnCompat == WarnCompat::On) {
     // Warn about policy versions for which support will be removed.
